android/app/build.gradle

```
@@ -10,17 +10,29 @@ plugins {
     id("org.jetbrains.kotlin.kapt") version "${kotlinVersion}"
 }
 
+def codegenPath = findNodeModulesPath("react-native-codegen", rootDir)
+def reactNativeDirPath = findNodeModulesPath("react-native", rootDir)
```

`react-native-codegen` should be resolved starting from `react-native` to ensure that we are picking up the correct version (e.g. Yarn could hoist another version of `react-native-macos` is on a different version):

```suggestion
def reactNativePath = file(findNodeModulesPath("react-native", rootDir))
def codegenPath = file(findNodeModulesPath("react-native-codegen", reactNativePath))
```